MAUI/Popup/getting-started.md

```diff
@@ -0,0 +1,234 @@
+                });
+
+            builder.ConfigureSyncfusionCore();
+            return builder.Build();
+        }
+    }
+}
+{% endhighlight %} 
+{% endtabs %}
+
+## Displaying popup
+
+You can display popup over your view by calling the `SfPopup.Show` method.
+
+Refer to the following code example for displaying popup.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:GettingStarted"
+             x:Class="GettingStarted.MainPage" 
+             Padding="0,40,0,0">
+     <StackLayout x:Name="mainLayout">
+       <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+               VerticalOptions="Start" HorizontalOptions="FillAndExpand"
+               Clicked="ClickToShowPopup_Clicked" />
+     </StackLayout>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+using Syncfusion.Maui.Popup;
+
+namespace GettingStarted
+{
+    public partial class MainPage : ContentPage
+    {
+        SfPopup popup;
+
+        public MainPage()
+        {
+            InitializeComponent();
+            popup = new SfPopup();
+        }
+
+        private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+        {
+            popup.Show();
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Popup with default appearance](Images/getting-started//maui-popup-with-default-appearance.png)
+
+You can download the source code of this sample [here](https://github.com/SyncfusionExamples/getting-started-.net-maui-popup).
+
+N> Two Popups cannot be displayed at the same time in a page.
+
+## Customize positioning
+
+The .NET MAUI Popup (SfPopup) allows showing the popup content at various positions.
+
+The following list of options are available to position the SfPopup in the desired position:
+
+* `Center Positioning`: Use the `SfPopup.IsOpen` property or `SfPopup.Show` method to display the SfPopup at the center.
+* `Absolute Positioning`: Use the `SfPopup.Show(x-position, y-position)` to display the SfPopup at the specified X and y position.
+* `Relative Positioning`: Use the `SfPopup.ShowRelativeToView(View, RelativePosition)` to display the SfPopup at any of the 8 positions relative to the specified view.
+* `Absolute relative positioning`: Use the `SfPopup.ShowRelativeToView(View, RelativePosition,x position,y position)` to display the SfPopup at an absolute x,y coordinate from the relative position of the specified view.
+
+## Customizing layouts
+
+By default, you can choose a layout from the following available layouts in the SfPopup by using the property `SfPopup.AppearanceMode`.
+
+* `OneButton`: Shows the SfPopup with one button in the footer view. This is the default value.
+* `TwoButton`: Shows the SfPopup with two buttons in the footer view.
```

Comment:
Add code snippet for setting AppearanceMode (Two Button)  and Screenshot.

Reply:
added screenshots and code snippet